server/xpub-server/entities/manuscript/resolvers/uploadManuscript.js

```diff
@@ -133,10 +133,6 @@ async function uploadManuscript(_, { file, id, fileSize }, { user }) {
     manuscript.files[fileIndex] = fileEntity
   }
 
-  // change the file entity from pending
-  fileEntity.type = 'MANUSCRIPT_SOURCE'
-  fileEntity.save()
```

Comment:
Is this a fix for the double uploading of a `MANUSCRIPT_SOURCE` or is this just reverting it back to how it was?